verifcid/allowlist.go

```diff
@@ -0,0 +1,69 @@
+package verifcid
+
+import (
+	mh "github.com/multiformats/go-multihash"
+)
+
+// DefaultAllowlist is the default list of hashes allowed in IPFS.
+var DefaultAllowlist Allowlist = defaultAllowlist{}
```

Comment:
```suggestion
var DefaultAllowlist defaultAllowlist
```
This prevents reassignments and the temptation to override this global for configuration (I think I got wrong originally or something, mb).

Reply:
Fixed

Reply:
> I think I got wrong originally

No, you were right originally.